### ISSUE TITLE
Unable to start Service

### ISSUE THREAD

**Issue by ContegoCode** [haram/watermark_disabler#2]
The project seems to be just what I am looking for to disable Test Mode watermark on the desktop without disabling Test signing or driver enforcement. I have no issues when trying to compile the driver file but when it comes to starting the service I am getting this message:

[SC] StartService FAILED 577:
Windows cannot verify the digital signature for this file. A recent hardware or software change might have installed a file that is signed incorrectly or damaged, or that might be malicious software from an unknown source.

I have compiled it as Release X64 and created the service with SC as can been seen by the error message above.

sc create "watermark_disabler" binPath="Path_to_sys" type=kernel

To recap on this issue I have debug set to on, no integrity checks set to on and also Test signing set to enabled by using CMD bcdedit command.

I am running windows 10 pro x64 with the correct SDK and WDK frameworks installed.
Any insight as to why I am getting this error would be greatly and truly apricated.

I have also tried this on a clean install with all options needed to run unsigned drivers but still get the same error.

Thank you for posting this and the research it took to make it possible.
